cmd/shortener/main.go

```diff
@@ -1,3 +1,36 @@
 package main
 
-func main() {}
+import (
+	"flag"
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"os"
+	"sanbright/go_shortener/internal/app/generator"
+	"sanbright/go_shortener/internal/app/handler"
+	"sanbright/go_shortener/internal/app/repository"
+	"sanbright/go_shortener/internal/app/service"
+	"sanbright/go_shortener/internal/config"
```

Comment:
Советую группировать импорты стандартной библиотеки, внутренних пакетов и импорты сторонних библиотек для улучшения читаемости.
```suggestion
	"flag"
	"net/http"
	"os"

	"sanbright/go_shortener/internal/app/generator"
	"sanbright/go_shortener/internal/app/handler"
	"sanbright/go_shortener/internal/app/repository"
	"sanbright/go_shortener/internal/app/service"
	"sanbright/go_shortener/internal/config"

	"github.com/gin-gonic/gin"
```